src/window/window.rs

```diff
@@ -409,6 +410,16 @@ impl Window {
                            mem::transmute(&mut out[0]));
         }
     }
+    
+    /// Get the current screen as an image
+    pub fn snap_image(&self) -> image::ImageBuffer<image::Rgb<u8>, Vec<u8>> {
```

Comment:
That's reasonable; I just did that, and also with `image::Rgb` and `image::imageops::flip_vertical`.

Any other suggestions?